Formula/librasterlite2.rb

```diff
@@ -0,0 +1,108 @@
+class Librasterlite2 < Formula
+  desc "Library to store and retrieve huge raster coverages"
+  homepage "https://www.gaia-gis.it/fossil/librasterlite2/index"
+  url "https://www.gaia-gis.it/gaia-sins/librasterlite2-sources/librasterlite2-1.1.0-beta1.tar.gz"
+  sha256 "f7284cdfc07ad343a314e4878df0300874b0145d9d331b063b096b482e7e44f4"
+  license any_of: ["MPL-1.1", "GPL-2.0-or-later", "LGPL-2.1-or-later"]
+
+  depends_on "pkg-config" => :build
+  depends_on "cairo"
+  depends_on "fontconfig"
+  depends_on "freetype"
+  depends_on "freexl"
+  depends_on "geos"
+  depends_on "giflib"
+  depends_on "jpeg"
+  depends_on "libgeotiff"
+  depends_on "libpng"
+  depends_on "librttopo"
+  depends_on "libspatialite"
+  depends_on "libspatialite"
+  depends_on "libtiff"
+  depends_on "lz4"
+  depends_on "minizip"
+  depends_on "openjpeg"
+  depends_on "pixman"
+  depends_on "proj@7"
```

Comment:
Do we not want this to depend on the latest `proj`?

Reply:
Should relate to dependency tree issue #94807.
From looks of it, this needs `proj@7` first and then has to be switched to `proj` in the other PR.

Reply:
Ah yes, `libspatialite` still uses `proj@7`.

Reply:
Yes exactly, once this is bottled and `spatiliate-gui` has been updated, it will be possible to migrate all `proj@7` dependents in https://github.com/Homebrew/homebrew-core/pull/94807 without any conflicts.